loggly/loggly.go

```diff
@@ -89,6 +90,8 @@ func (l *Adapter) SendMessage(msg logglyMessage) error {
 	url := fmt.Sprintf("%s%s/%s", logglyAddr, logglyEventEndpoint, l.token)
 	req, err := http.NewRequest("POST", url, bytes.NewBuffer(js))
 
+	req.Header.Add("X-LOGGLY-TAG", os.Getenv(logglyTagsEnvVar))
```

Comment:
We have to reach out to the environment every time we make a request. Though this may not be slow, it's still not efficient.

I would lean towards grabbing the logglyTagsEnvVar in NewLogglyAdapter and adding a field to Adapter and in this line only add the header if that field is not an empty string.